frontend/components/site/siteBuildsBuild.jsx

```diff
@@ -0,0 +1,327 @@
+              { React.createElement(icon) }
+            </span>
+            )}
+            { status }
+          </h3>
+        </div>
+        <div className="build-info-details">
+          <span id={`build-${build.id}`} className="usa-sr-only">
+            Build #
+            {build.id}
+          </span>
+          {build.startedAt && !!done && (
+          <p>
+            Finished
+            {' '}
+            {' '}
+            <span title={dateAndTimeSimple(build.startedAt)}>
+              { timeFrom(build.startedAt) }
+            </span>
+          </p>
+          )}
+          <p>
+            { (done && !!build.startedAt) && (
+            <>
+              {messagePrefix}
+
+              { duration(build.startedAt, build.completedAt) }
+            </>
+            )}
+            {(!done) && (
+            <>
+              {messagePrefix}
+
+              { duration(build.createdAt, build.completedAt) }
+            </>
+            )}
+          </p>
+
+          <p className="build-info-logs-link">
+            {build.startedAt ? buildLogsLink(build.id) : ''}
+          </p>
+        </div>
+
+      </th>
+      <td data-title="Branch">
+        <div className="branch-info">
+          { branchLink(build) }
+          <div className="commit-info">
+            { shaLink(build) }
+            <span className="commit-user" title={build.user?.email}>
+              { build.username }
+            </span>
+            <span className="commit-time" title={dateAndTime(build.createdAt)}>
+              { timeFrom(build.createdAt) }
+            </span>
+          </div>
+        </div>
+        { isPreviewBuild(build) && build.state === 'success'
+          && (
+            <p className="site-link">
+              <BranchViewLink
+                branchName={build.branch}
+                site={build.site}
+                showIcon
+                completedAt={build.completedAt}
+              />
+            </p>
+          )}
+      </td>
+      { showBuildTasks && (
+        <td data-title="Scans" className="">
+          <ScanResultsSummary {...summarizeTaskResults(build, assumeScanQueued)}>
+            { summarizeTaskResults(build)?.status !== 'cancelled' && summarizeTaskResults(build)?.status !== 'queued' && !assumeScanQueued && (
+              <>
+                {buildHasBuildTasks(build) && !buildTasksInProgress(build) && (
+                  <p>
+                    <span title={dateAndTimeSimple(build.BuildTasks[0].updatedAt)}>
+                      { timeFrom(build.BuildTasks[0].updatedAt) }
+                    </span>
+                  </p>
+                )}
+                <p className="scan-link">
+                  <Link to={`/sites/${siteId}/builds/${build.id}/scans`}>View scans</Link>
+                </p>
+              </>
+            )}
+          </ScanResultsSummary>
+        </td>
+      )}
+      <td data-title="Actions" className="table-actions">
```

Comment:
When clicking the "Run scans" button, there is no feedback to the users that it was clicked and the request was successful or not. A user can click this many times in a row and it kicks off many scan without throttling or rate limiting the user.